phone/src/apps/messages/components/ui/MessageEmbed.tsx

```diff
@@ -69,10 +89,16 @@ const LocationEmbed = ({ embed }: { embed: Location; isMine: boolean }) => {
     });
   };
 
+  const showMessageAsTypo = message === t('MESSAGES.LOCATION_MESSAGE');
+
   return (
     <StyledMessage>
       <Box>
-        <Typography>{t('MESSAGES.LOCATION_MESSAGE')}</Typography>
+        {showMessageAsTypo ? (
```

Comment:
Shouldn't this always be typography

Reply:
I only used Typography when the message is the default one, so that when the message is custom, it'll show like the other messages do:
![image](https://user-images.githubusercontent.com/27494805/167251481-220b526d-66ca-45b7-880c-d19fcef03367.png)
I can change it if you prefer

Reply:
I think it should always match for the location sharing otherwise it doesn't quite look right

Reply:
Ok I'll edit it in next commit

Reply:
Changed in last commit